From 5861817bb21f6f7174067aac09184657f4e8f60f Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 10 Nov 2016 15:36:06 -0600
Subject: [PATCH] debian: -fno-PIE

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 debugss_module/debugss-mod/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debugss_module/debugss-mod/Makefile b/debugss_module/debugss-mod/Makefile
index 2c41fe7..8169277 100644
--- a/debugss_module/debugss-mod/Makefile
+++ b/debugss_module/debugss-mod/Makefile
@@ -16,7 +16,7 @@
 #
 
 obj-m := debugss_kmodule.o
-CFLAGS_debugss_kmodule.o := -D$(PLATFORM)
+CFLAGS_debugss_kmodule.o := -D$(PLATFORM) -fno-PIE
 
 KVERSION ?= $(shell uname -r)
 KERNEL_SRC  ?= /lib/modules/$(KVERSION)/build
-- 
2.10.2

